### DISCUSSION TITLE
Question: How to change Lotus root verifier?

### DISCUSSION THREAD

**Discussion by Schwartz10** [filecoin-project/lotus#3440] [Storage Provider] [answered]
I'm trying to run a local network where I can add verifiers, and verify additional clients via the `lotus-shed` cli. There are a couple hardcoded addresses that I'm not sure how to change in order to get this to work locally:

[lotus/chain/gen/genesis/t06_vreg.go](https://github.com/filecoin-project/lotus/blob/master/chain/gen/genesis/t06_vreg.go#L21)
[lotus/cmd/lotus-shed/verifreg.go](https://github.com/filecoin-project/lotus/blob/master/cmd/lotus-shed/verifreg.go#L40)

Any pointers would be appreciated!

**Comment by whyrusleeping** [comment, marked as answer]
@Schwartz10 you need to set them in your `devnet.json` file before you start your lotus node up.

This segment looks like this:
```json
  "VerifregRootKey": {
    "Type": "multisig",
    "Balance": "0",
    "Meta": {
      "Signers": [
        "t080"
      ],
      "Threshold": 1,
      "VestingDuration": 0,
      "VestingStart": 0
    }
  }
```

Just change the signers to whatever addresses you like (and set other parameters as you want)
